ce/api/util.py

```diff
@@ -95,8 +96,15 @@ def mean_datetime(datetimes):
     mean = np.mean(timestamps)
     return datetime.fromtimestamp(mean, tz=timezone.utc)
 
+def validate_cell_method(cell_method):
```

Comment:
This is very explicit, which in this case I like :) It would save a couple lines until we support methods, but you could probably also write the function body as:
```
return cell_method in ('mean', 'standard_deviation')
```
and it would be equally explicit, but also short.